digits/job.py

```diff
@@ -324,6 +325,12 @@ def wait_completion(self):
         if hasattr(self, 'event'):
             self.event.wait()
 
+    def delete_timeout(self):
+        """
+        Returns how long a non-persistent job should exist for
+        """
+        return NON_PERSISTENT_JOB_DELETE_TIMEOUT_SECONDS
```

Comment:
Wouldn't it be simpler to make this a class attribute? `self.DELETE_TIMEOUT` instead of `self.delete_timeout()`?